src/client/client4.ts

```diff
@@ -3307,51 +3311,62 @@ export default class Client4 {
     // Cloud routes
     getCloudProducts = () => {
         return this.doFetch<Product[]>(
-            `${this.getBaseRoute()}/cloud/products`, {method: 'get'},
+            `${this.getCloudRoute()}/products`, {method: 'get'},
         );
     };
 
     createPaymentMethod = async () => {
         return this.doFetch(
-            `${this.getBaseRoute()}/cloud/payment`,
+            `${this.getCloudRoute()}/payment`,
             {method: 'post'},
         );
     }
 
     getCloudCustomer = () => {
         return this.doFetch<CloudCustomer>(
-            `${this.getBaseRoute()}/cloud/customer`, {method: 'get'},
+            `${this.getCloudRoute()}/customer`, {method: 'get'},
         );
     }
 
     updateCloudCustomer = (customerPatch: CloudCustomerPatch) => {
         return this.doFetch<CloudCustomer>(
-            `${this.getBaseRoute()}/cloud/customer`,
+            `${this.getCloudRoute()}/customer`,
             {method: 'put', body: JSON.stringify(customerPatch)},
         );
     }
 
     updateCloudCustomerAddress = (address: Address) => {
         return this.doFetch<CloudCustomer>(
-            `${this.getBaseRoute()}/cloud/customer/address`,
+            `${this.getCloudRoute()}/customer/address`,
             {method: 'put', body: JSON.stringify(address)},
         );
     }
 
     confirmPaymentMethod = async (stripeSetupIntentID: string) => {
         return this.doFetch(
-            `${this.getBaseRoute()}/cloud/payment/confirm`,
+            `${this.getCloudRoute()}/payment/confirm`,
             {method: 'post', body: JSON.stringify({stripe_setup_intent_id: stripeSetupIntentID})},
         );
     }
 
     getSubscription = () => {
         return this.doFetch<Subscription>(
-            `${this.getBaseRoute()}/cloud/subscription`,
+            `${this.getCloudRoute()}/subscription`,
+            {method: 'get'},
+        );
+    }
+
+    getInvoices = () => {
+        return this.doFetch<Invoice[]>(
+            `${this.getCloudRoute()}/subscription/invoices`,
```

Comment:
We have to pass the subscription id like `/subscription/${subscriptionId}/invoices` . My main concern on this is that we have to pass the stripe subscription id, eg. `sub_IEtgAkaTisBQTG`, because we don't store the subscription information

// @marianunez

Reply:
We shouldn't need to do this actually, if we mimic what I've done for getting the subscription (ie. using the installation ID) then we can avoid this.

Reply:
Correct, the subscription id will be passed in the enterprise by looking at the env var for installation id. It will be transparent from the front-end side.

Reply:
Understood :)